tool/loadgen/main.go

```diff
@@ -121,7 +121,7 @@ func test(containerName string) TestResult {
 		Stats:   map[string]Stats{"app": stats},
 	}
 	if hasRelay {
-		tr.RelayMetrics = relayMetrics()
+		tr.RelayMetrics = getRelayMetrics()
```

Comment:
It is not very common to write "getFoo()" in Go, where "foo()" would do. https://golang.org/doc/effective_go#Getters